test/test-tablet-validity.c

```diff
@@ -169,6 +169,9 @@ test_vidpid(gconstpointer data)
 			g_assert_cmpint(libwacom_get_product_id(device), >=, 0);
 			break;
 		case WBUSTYPE_USB:
+			if (libwacom_get_vendor_id(device) == 0x056A)
+				g_assert_cmpint(libwacom_get_product_id(device), !=, 0x84);
+			/* fall through */
```

Comment:
Please put a comment here that `0x84` is the wireless dongle so others don't have to go through `git blame` to figure that out.